Updated docs for Raspbian OS (#14816)
openvinotoolkit/openvino: docs/install_guides/installing-openvino-raspbian.md

@sphinxdirective
.. note::

>   * These steps apply to Raspbian OS, which is an official OS for Raspberry Pi boards.

Reviewer: ```suggestion
  * These steps apply to Raspbian OS (the official OS for Raspberry Pi boards).
```
